modules/cluster-logging-collector-syslog.adoc

```diff
@@ -0,0 +1,184 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-external.adoc
+
+[id="cluster-logging-collector-syslog_{context}"]
+= Sending logs using the Fluentd syslog plug-in (RFC 3164)
+
+You can use the Fluentd *syslog* plug-in to send a copy of your logs to an external syslog server, 
+not the default Elasticsearch. Note the following about the *syslog* plug-in:
+
+* uses syslog protocol (RFC 3164), not RFC 5424;
+* does not support TLS and thus, is not secure;
+* does not provide Kubernetes metadata, systemd data, or other metadata.
+
+[NOTE]
+====
+The Fluentd *syslog* plug-in method is deprecated and will be removed in a future release.
+====
+
+There are two versions of the Fluentd *syslog* plug-in:
+
+* *out_syslog*. The non-buffered implementation, which communicates through UDP, does not buffer data and writes out results immediately. 
```

Comment:
s/*out_syslog*./*out_syslog*:/

Reply:
One of the many such inconsistencies in the docs. 
*out_syslog*. 
*out_syslog.*
*out_syslog*:
*out_syslog* - 
*out_syslog*